CONTRIBUTING.md

```diff
@@ -43,3 +46,39 @@ make test-integration
 * Commits should be as small as possible, while ensuring that each commit is correct independently.
 
 * Add tests relevant to the fixed bug or new feature.
+
+
+## Troubleshooting build
+
+### docker approach
+you can build in a docker container by going to root dir of gopass project and:
+```
+docker run --rm -v "$PWD":/go/src/github.com/justwatchcom/gopass -w /go/src/github.com/justwatchcom/gopass golang:stretch go build -v
+```
+
+replace `go build -v` in above command with `make test` or anything else you'd like to run inside docker.
+
+you can also get an interactive shell inside the container via:
+```
+docker run -it -v "$PWD":/go/src/github.com/justwatchcom/gopass -w /go/src/github.com/justwatchcom/gopass golang:stretch bash
+```
+in which you can run `go build`, `make test` or whatever you'd like
+
+### non-docker
+
+make sure $GOPATH and $GOROOT are set correctly.
+$GOROOT should be the go dir where VERSION file is, and which has bin/ pkg/ and src/ as sub dirs.
+$GOPATH is roughly where your go projects src code is.
+`go env` will show you the current dirs used.
+
+heres a quick start:
+- `mkdir $HOME/go-workspace`
```

Comment:
I wouldn't suggest to set `GOPATH=$HOME/go-workspace`, we should recommend the default of `GOPATH=$HOME/go`.